tests/playwright/account.spec.js

```diff
@@ -0,0 +1,41 @@
+import { test, expect } from "@playwright/test";
+import { AccountPage } from "./pages/AccountPage";
+import { RegisterPage } from "./pages/RegisterPage";
+import { BasePage } from "./pages/BasePage";
+
+test("Register account", async ({ page }) => {
```

Comment:
Maybe we can use the registration tests from here: https://github.com/rapidez/account/pull/73?